_data/simple-icons.json

```diff
@@ -8200,7 +8200,7 @@
         },
         {
             "title": "Signal",
-            "hex": "2592E9",
+            "hex": "3977F0",
             "source": "https://play.google.com/store/apps/details?id=org.thoughtcrime.securesms"
```

Comment:
The source should be https://github.com/signalapp/Signal-Desktop/blob/9db8765b6cf270195e45a7f251374d4e53d54c95/images/signal-logo.svg

Reply:
on it!